sdk/search/Azure.Search.Documents/tests/Batching/BatchingTests.cs

```diff
@@ -191,6 +191,10 @@ public override async Task<Response<IndexDocumentsResult>> IndexDocumentsAsync<T
                     RaiseNotification();
                 }
             }
+
+            public virtual SearchIndexingBufferedSender<T> CreateIndexingBufferedSender<T>(
+                SearchIndexingBufferedSenderOptions<T> options = null) =>
+                new SearchIndexingBufferedSender<T>(this, options);
```

Comment:
We'd typically go about this by changing all the `private` members of `SearchClient` to `internal virtual`.  This is less code churn before a release so I think it's fine too.